src/main/java/chatbot/Parser.java

```diff
@@ -0,0 +1,122 @@
+package chatbot;
+
+import task.Deadline;
+import task.Event;
+import task.TaskList;
+import task.Todo;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.time.format.DateTimeParseException;
+import java.time.format.ResolverStyle;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.time.format.ResolverStyle;
```

Comment:
Should these java imports be at the top of all the imports instead? and should the imports also be in lexicographic order instead?